qnm/angular.py

```diff
@@ -105,7 +128,23 @@ def M_matrix_elem(s, c, m, l, lprime):
 
 # I don't know if this is necessary ... can just iterate
 def give_M_matrix_elem_ufunc(s, c, m):
-    """ TODO Document """
+    """ 
```

Comment:
First line of docstring should be a one-sentence (preferably one-line) description of the method, e.g. "Give a ufunc that implements matrix elements of the spherical-spheroidal decomposition matrix"